### DISCUSSION TITLE
Improve writing experience with autocomplete

### DISCUSSION THREAD

**Discussion by swijckmans** [avitorio/outstatic#111] [Ideas]
Project like https://novel.sh/ are becoming more common and allow for a better writing experience. Would it be hard to use these to improve Outstatic?

**Comment by avitorio** [comment]
Hey, I was checking novel.sh over the weekend and am considering making the switch to use it. We both use TipTap as the underlying editor framework, so I think it wouldn't be too much of a hassle. @steven-tey is a beast.

Thanks for the suggestion ❤️

**Comment by josebenitezg** [comment]
Thank could be an awesome feature

**Reply by avitorio** [reply]
There's an active branch working on this. I'll push this in the near future.

**Reply by josebenitezg** [reply]
awesome! let me know if I can help with something! This will be a gamechanger for the CMS

**Reply by avitorio** [reply]
Thanks @josebenitezg help is always welcome. I'll create a PR for this and will tag you for help.

**Reply by josebenitezg** [reply]
Amazing! Let's put Outstatic to the next level 😎
